libs/feature/editor/src/lib/components/overview-upload/overview-upload.component.ts

```diff
@@ -31,40 +50,98 @@ export class OverviewUploadComponent implements OnInit {
   ngOnInit(): void {
     this.recordsApiService
       .getAllResources(this.metadataUuid)
-      .subscribe((resources) => {
-        this.resourceFileName = resources[0]?.filename
-        this.resourceUrl = resources[0]?.url
-        this.cd.markForCheck()
+      .pipe(takeUntil(this.destroy$))
```

Comment:
I don't think you need that; the `getAllResources` observable is an HTTP call, so it is short-lived: as soon as the result is emitted, it also completes.